### DISCUSSION TITLE
Integrating OpenAI's gpt-3.5-turbo-0613 into Flowise: Unlocking New Possibilities

### DISCUSSION THREAD

**Discussion by netanelyasi** [FlowiseAI/Flowise#403] [Q&A]
Firstly, I would like to express my gratitude for your incredible tool. It has truly been a game-changer in my workflow and has greatly contributed to enhancing the effectiveness of my operations.

Recently, OpenAI has released an update to their API: gpt-3.5-turbo-0613. As a loyal user of your tool, I would be thrilled if this new feature could be integrated into the Flowise environment. Given the past and ongoing evolution of AI technologies, I believe the inclusion of gpt-3.5-turbo-0301 would enhance the functionalities of Flowise and continue to maintain its high standards.

Currently, I am using Flowise in conjunction with render.com, and I haven't found a way to incorporate the updated OpenAI version into the system. It would be extremely helpful if you could provide information on whether there are any plans to update Flowise to include gpt-3.5-turbo-0613. If such an update is planned or already available, could you kindly share the necessary steps to apply this update to my existing setup?

I am very much looking forward to using the new features and capabilities that come with the gpt-3.5-turbo-0613 update.

I appreciate your attention to this matter and thank you in advance for your assistance.

Best regards,
Netanel

**Comment by HenryHengZJ** [comment]
answered here - https://github.com/FlowiseAI/Flowise/issues/402
